public/images/pro-landing/flag-uk.svg

```diff
@@ -0,0 +1 @@
.86C4.784,17.629,4.412,17.8,4.059,18H2.709c0.052-.029.1-0.06,0.156-0.087l11.646-5.823c0.042-.021.084-0.044,0.125-0.066L14.63,12H14.488c-0.765,0-1.53,0-2.295,0a0.729,0.729,0,0,0-.343.08Q6.014,15,.176,17.92A0.306,0.306,0,0,0,.038,18c-0.062-.027-0.028-0.081-0.028-0.12,0-.64,0-1.28,0-1.92a0.163,0.163,0,0,0,.146-0.047Q3.984,14,7.811,12.089c0.041-.021.082-0.044,0.162-0.087H0.008V10.8Z"/><path class="cls-2" d="M36.015,7.2Q28,7.2,19.979,7.2c-0.135,0-.171-0.032-0.17-0.169q0.007-3.518,0-7.035h1.2V5.483c0.065-.03.105-0.047,0.144-0.066q4.859-2.429,9.717-4.86c0.365-.183.737-0.351,1.09-0.557h1.35c-0.052.029-.1,0.06-0.156,0.087L21.512,5.91c-0.042.021-.083,0.044-0.125,0.066L21.392,6h0.142c0.765,0,1.53,0,2.295,0a0.729,0.729,0,0,0,.343-0.08Q30.009,3,35.847.083A0.306,0.306,0,0,0,35.985,0c0.062,0.027.028,0.081,0.028,0.12,0,0.64,0,1.28,0,1.92a0.163,0.163,0,0,0-.146.047Q32.039,4,28.211,5.914C28.17,5.935,28.13,5.958,28.05,6h7.965V7.2Z"/><path class="cls-3" d="M21.012,18V12.547a0.14,0.14,0,0,1,.131.034Q26.5,15.26,31.864,17.942c0.035,0.017.066,0.041,0.1,0.061H21.012Z"/><path class="cls-3" d="M4.059,18c0.353-.205.725-0.374,1.09-0.557q4.858-2.432,9.717-4.86c0.039-.02.08-0.037,0.144-0.066V18H4.059Z"/><path class="cls-3" d="M31.964,0c-0.353.205-.725,0.374-1.09,0.557q-4.858,2.432-9.717,4.86c-0.039.02-.08,0.037-0.144,0.066V0H31.964Z"/><path class="cls-3" d="M15.011,0V5.456a0.14,0.14,0,0,1-.131-0.034Q9.519,2.743,4.159.061C4.124,0.044,4.092.021,4.059,0H15.011Z"/><path class="cls-1" d="M35.985,0a0.306,0.306,0,0,1-.138.083Q30.01,3,24.172,5.923a0.729,0.729,0,0,1-.343.08c-0.765,0-1.53,0-2.295,0H21.392l-0.006-.025c0.042-.022.083-0.045,0.125-0.066L33.158,0.087c0.053-.027.1-0.058,0.156-0.087h2.67Z"/><path class="cls-1" d="M0.038,18A0.306,0.306,0,0,1,.176,17.92Q6.013,15,11.851,12.08A0.729,0.729,0,0,1,12.194,12c0.765,0,1.53,0,2.295,0H14.63l0.006,0.025c-0.042.022-.083,0.045-0.125,0.066L2.865,17.917c-0.053.027-.1,0.058-0.156,0.087H0.038Z"/><path class="cls-3" d="M0.008,2.04a0.112,0.112,0,0,1,.119.033Q4,4.009,7.878,5.947C7.9,5.956,7.912,5.967,7.97,6H0.008V2.04Z"/><path class="cls-3" d="M36.015,6H28.05c0.08-.043.12-0.066,0.162-0.087Q32.04,4,35.869,2.088a0.163,0.163,0,0,1,.146-0.047V6Z"/><path class="cls-3" d="M0.008,12H7.973c-0.08.043-.12,0.066-0.162,0.087Q3.983,14,.154,15.916a0.163,0.163,0,0,1-.146.047V12Z"/><path class="cls-3" d="M36.015,15.963a0.112,0.112,0,0,1-.119-0.033q-3.876-1.935-7.751-3.873c-0.017-.009-0.034-0.02-0.091-0.054h7.961v3.961Z"/><path class="cls-1" d="M35.985,18a0.365,0.365,0,0,0-.151-0.09L24.2,12.1l-0.142-.072A0.19,0.19,0,0,1,24.192,12c0.78,0,1.56,0,2.339,0a0.669,0.669,0,0,1,.314.074q4.509,2.258,9.02,4.512c0.049,0.024.1,0.044,0.15,0.066,0,0.42,0,.84,0,1.26C36.012,17.942,36.049,17.991,35.985,18Z"/><path class="cls-1" d="M0.038,0A0.365,0.365,0,0,0,.189.09L11.822,5.908,11.964,5.98a0.192,0.192,0,0,1-.133.02c-0.78,0-1.56,0-2.339,0a0.669,0.669,0,0,1-.314-0.074Q4.669,3.67.158,1.416C0.109,1.392.058,1.372,0.008,1.35c0-.421,0-0.841,0-1.261C0.01,0.061-.026.011,0.038,0Z"/></g></svg>
```

Comment:
Oops! That's actually super funny because I sized the UK flag down using a different flag file than even Australia 😂